.github/workflows/ruby-ci.yml

```
@@ -0,0 +1,32 @@
+name: Ruby-CI
+
+on:
+  push:
+  branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  ruby-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout
+
+      - uses: ruby/setup-ruby@v1
+        name: Use Ruby 2.6
+        with:
+          ruby-version: 2.6
+          bundler-cache: true
+      - name: 📦 Install dependencies
+        run: |
+          bundle config set ignore_messages true
+          bundle install
```

`bundle install --clean` ran into various issues, wondering if this is okay

---

I think it's fine 👍 I think we should add `--frozen` so we are sure the Gemfile is not updated.
```suggestion
          bundle install --forzen
```